src/android/FileTransferBackground.java

```diff
@@ -81,7 +82,7 @@ private void sendProgress(final String id, int progressPercent) {
         }
     }
 
-    private void sendSuccess(final String id, final String response, int statusCode) {
+    private void sendSuccess(final String id, final String response, int statusCode, long uploadDuration, long finishUploadTime) {
```

Comment:
We should pass these as an object now as Code Climate is giving some warnings: https://codeclimate.com/github/spoonconsulting/cordova-plugin-background-upload/pull/282

Reply:
yes we have too many params i think too